node/pkg/driver/device_connectivity/device_connectivity_fc_helper_test.go

```diff
@@ -0,0 +1,350 @@
+/**
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package device_connectivity_test
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	gomock "github.com/golang/mock/gomock"
+	mocks "github.com/ibm/ibm-block-csi-driver/node/mocks"
+	"github.com/ibm/ibm-block-csi-driver/node/pkg/driver/device_connectivity"
+)
+
+func TestHelperWaitForPathToExist_Fc(t *testing.T) {
```

Comment:
again if its the same as iscsi, then you can avoid test it here if it already tested in the iscsi, just refactor to generic scsi and have one UT

Reply:
ok, refactor it

Reply:
status?